can/SConscript

```diff
@@ -3,18 +3,13 @@ Import('env', 'envCython', 'cereal')
 import os
 from opendbc.can.process_dbc import process
 
-dbcs = []
-for x in sorted(os.listdir('../')):
-  if x.endswith(".dbc"):
-    def compile_dbc(target, source, env):
-      process(source[0].path, target[0].path)
-    in_fn = [os.path.join('../', x), 'dbc_template.cc']
-    out_fn = os.path.join('dbc_out', x.replace(".dbc", ".cc"))
-    dbc = env.Command(out_fn, in_fn, compile_dbc)
-    env.Depends(dbc, ["dbc.py", "process_dbc.py"])
-    dbcs.append(dbc)
-
-libdbc = env.SharedLibrary('libdbc', ["dbc.cc", "parser.cc", "packer.cc", "common.cc"]+dbcs, LIBS=["capnp", "kj"])
+envDBC = env.Clone()
+dbc_file_path = '-DDBC_FILE_PATH=\'"%s"\'' % (envDBC.Dir("..").abspath)
+if isinstance(envDBC['CXXFLAGS'], list):
+  envDBC['CXXFLAGS'] += [dbc_file_path]
+else:
+  envDBC['CXXFLAGS'] += ' ' + dbc_file_path
+libdbc = envDBC.SharedLibrary('libdbc', ["dbc.cc", "parser.cc", "packer.cc", "common.cc"], LIBS=["capnp", "kj"])
```

Comment:
Think this is the cleanest, then you override the BASEDIR envvar if you want to get the DBCs from somewhere else (like PJ would use). If we use RPATH we'd have to get the path of the shared library in the CANParser and CANPacker Cython libs, then pass as an argument to libdbc